api/notd/model.py

```diff
@@ -69,3 +69,4 @@ class RegistryToken:
     collectionImageUrl: Optional[str]
     collectionOpenSeaUrl: Optional[str]
     collectionExternalUrl: Optional[str]
+    
```

Comment:
fix pls

Reply:
The lint fails without a newline, is there something i am missing?